File: src/receptor.Application/Recipes/DTOs/CreateRecipeDto.cs
namespace receptor.Application.Recipes.DTOs;

public class CreateRecipeDto
{
    public required string Name { get; set; }
    public required string UserUuid { get; set; }
    public required string Description { get; set; }
    public int Time { get; set; }
    public required string[] Ingredients { get; set; }
    public required string[] Process { get; set; }
    public string? ImagePath { get; set; }
}

File: src/receptor.client/src/Types/CreateRecipeDto.ts
export type CreateRecipeDto = {
    name: string;
    description: string;
    userUuid: string | null;
    imagePath: string;
    time: number;
    ingredients: string[];
    process: string[];
};

export type CreateRecipeRequest = {
    recipeDto: CreateRecipeDto;
};

